apiserver/worker/backend.py

```diff
@@ -3,7 +3,11 @@
 from hashlib import md5
 import json
 import os
-from time import gmtime, strftime
+from time import gmtime, strftime, sleep
+
+
+# Needs to match corresponding value in apiserver/config.py
+MAX_BOT_UPLOAD_SIZE = 100 * 1024 * 1024
```

Comment:
Rather than define it twice can't wejust pick it from there/another central location for both?

Reply:
Yeah, the issue is that the worker doesn't assume it has access to the server codebase (though this happens to be true currently). Perhaps we can make it part of the `config.json`, or have a server endpoint that returns the value?

Reply:
Config.json would be fine :)